tests/console/powerpc_utils.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: powerpc-utils
+# Summary: regression test powerpc-utils, verify that powerpc-utils works as expected
+# Maintainer: Zaoliang Luo <zluo@suse.com>
+
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    zypper_call('refresh');
+    zypper_call('in powerpc-utils');
+    # check installed or updated power-utils version
+    assert_script_run("rpm -q powerpc-utils");
+
+    # run powerpc-utils and save logs
+    my $logs = "/tmp/powerpc-utils.log";
+    assert_script_run("echo -e '__________ lparstat __________\n\n\n' > $logs");
```

Comment:
If you use single quotes on the outside and double quotes on the inside, it might not show things like https://openqa.suse.de/tests/5470382#step/powerpc_utils/29 in red.

```suggestion
    assert_script_run('echo -e "__________ lparstat __________\n\n\n" > '.$logs);
```

Note that I moved `$logs` outside of the single quotes.

Reply:
yes, thanks. updated now and new test run looks good as well:
https://progress.opensuse.org/issues/49025